README.md

```diff
@@ -48,7 +48,7 @@ cXT200 (pistachio) - the name of the Imagination Technologies SoC that Ci40 uses
 1. Copy base config
 
     ```
-    $ cat target/linux/pistachio/creator-platform-default-cascoda.config > .config
+    $ cat target/linux/pistachio/creator-platform-default.config > .config
```

Comment:
I think we should rename the file to "marduk-platform-default.config", as the creator term in the config filename has got no real context to it.  So, the filename should be
* target/linux/pistachio/marduk-platform-default.config
* target/linux/pistachio/marduk-platform-legacy.config

I also think there is no need of "default" in legacy config filename